src/coffea/dataset_tools/dataset_query.py

```diff
@@ -578,16 +600,17 @@ def load_dataset_definition(
         self.do_sites_filters(ask_clear=False)
         print("Getting replicas")
         if replicas_strategy == "manual":
-            self.do_replicas(mode=None, selection="all")
+            out_replicas = self.do_replicas(mode=None, selection="all")
         else:
-            if replicas_strategy not in ["round-robin", "choose"]:
+            if replicas_strategy not in ["round-robin", "choose", "first"]:
                 print(
-                    "Invalid replicas-strategy: please choose between manual|round-robin|choose"
+                    "Invalid replicas-strategy: please choose between manual|round-robin|choose|first"
                 )
                 exit(1)
-            self.do_replicas(mode=replicas_strategy, selection="all")
+            out_replicas = self.do_replicas(mode=replicas_strategy, selection="all")
```

Comment:
If I'm seeing correctly, `do_replicas()` doesn't return anything so `out_replicas` will be `None`